dist/bin/reacli.js

```diff
@@ -159,6 +160,34 @@ var dismissFlowOption = function dismissFlowOption(dumbString, containerString)
 	return [dumbString, containerString];
 };
 
+// REDUX
+var reduxContainerFlags = {
+	"redux-import-connect": "import { connect } from 'react-redux';",
+	"redux-map-dispatch-to-props": "const mapDispatchToProps = dispatch => ({\n\t// action: (input) => dispatch(action(input)),\n});",
+	"redux-map-state-to-props": "const mapStateToProps = () => ({}); // or (state) => ({});"
+};
+
+var applyReduxOption = function applyReduxOption(containerString, componentName) {
+	for (var key in reduxContainerFlags) {
+		containerString = containerString.replace(new RegExp("\\[\\[" + key + "\\]\\]", "u"), reduxContainerFlags[key]);
+	}
+	containerString = containerString.replace(new RegExp("\\[\\[redux-connection\\]\\]" + componentName + "Container", "u"), "connect(mapStateToProps, mapDispatchToProps)(" + componentName + "Container)");
```

Comment:
if you use [[something]] patterns all over the place it really makes sense to have a function dedicated for the job. But really you should use some already built solution from npm. Focus on what's really specific for your app, use ready solutions for everything else.

Reply:
found one such module https://www.npmjs.com/package/matchstick
but too few downloads

Reply:
https://www.npmjs.com/package/string-template

Reply:
https://www.npmjs.com/package/string-placeholder